bot/cogs/moderation.py

```diff
@@ -181,12 +198,137 @@ async def clear(self, ctx: Context, amount: int) -> None:
                 **Amount**: {amount}
                 """
             ),
-            color=discord.Color.orange(),
+            color=Color.orange(),
         )
         message = await ctx.send(ctx.author.mention, embed=embed)
         await asyncio.sleep(2.5)
         await message.delete()
 
+    @command()
+    @has_permissions(manage_messages=True)
+    async def shift(self, ctx, count: int, target: TextChannel, copy: bool = False) -> None:
+        """Copy or Move specified messages amount to specified channel"""
+        messages = []
```

Comment:
```suggestion
        """Copy or Move specified messages amount to specified channel"""
        if not (5 <= count <= 100):
            await ctx.send("Amount of messages shifted must be greater than 0 and smaller than 100")
        messages = []
```
I suggest this because otherwise this command may end up being rather spammy due to rate-limiting